Buildings/Experimental/DHC/CentralPlants/Cooling/Plant.mo

```diff
@@ -0,0 +1,383 @@
+  connect(pumCW.P,totPPum.u[1:2])
+    annotation (Line(points={{41,94},{80,94},{80,140},{240,140},{240,160.5},{258,160.5}},color={0,0,127}));
+  connect(pumCHW.P,totPPum.u[3:4])
+    annotation (Line(points={{-29,-2},{-20,-2},{-20,44},{84,44},{84,136},{242,136},
+          {242,158.5},{258,158.5}}, color={0,0,127}));
+  connect(totPFan.y,PFan)
+    annotation (Line(points={{282,200},{320,200}},color={0,0,127}));
+  connect(cooTowWitByp.PFan,totPFan.u[1:2])
+    annotation (Line(points={{-19,96},{-6,96},{-6,200},{258,200},{258,199}},color={0,0,127}));
+  connect(totPCoo.y,PCoo)
+    annotation (Line(points={{282,240},{320,240}},color={0,0,127}));
+  connect(mulChiSys.P,totPCoo.u[1:2])
+    annotation (Line(points={{-11,2},{-14,2},{-14,40},{234,40},{234,239},{258,239}},color={0,0,127}));
+  connect(mulChiSys.port_b2,splCHWSup.port_1)
+    annotation (Line(points={{10,-6},{80,-6},{80,-80}},color={0,127,255}));
+  connect(splCHWSup.port_3,senTCHWSup.port_a)
+    annotation (Line(points={{90,-90},{90,-40},{140,-40}},
+                                                 color={0,127,255}));
+  connect(splCHWSup.port_2,senMasFloByp.port_a)
+    annotation (Line(points={{80,-100},{80,-110},{60,-110},{60,-90},{40,-90}},color={0,127,255}));
+  connect(joiCHWRet.port_2,pumCHW.port_a)
+    annotation (Line(points={{-80,-80},{-80,-6},{-50,-6}},color={0,127,255}));
+  connect(joiCHWRet.port_1,senMasFlo.port_b)
+    annotation (Line(points={{-80,-100},{-80,-110},{-100,-110},{-100,-90},{-170,-90}},color={0,127,255}));
+  connect(joiCHWRet.port_3,valByp.port_b)
+    annotation (Line(points={{-70,-90},{-40,-90}},color={0,127,255}));
+  connect(expTanCHW.ports[1],pumCHW.port_a)
+    annotation (Line(points={{-90,-30},{-80,-30},{-80,-6},{-50,-6}},color={0,127,255}));
+  connect(senTCHWRet.T,chiStaCon.TChiWatRet)
+    annotation (Line(points={{-260,-79},{-260,196},{-242,196}},color={0,0,127}));
+  connect(senTCHWSup.T,chiStaCon.TChiWatSup)
+    annotation (Line(points={{150,-51},{150,-120},{-220,-120},{-220,166},{-252,166},
+          {-252,192},{-242,192}},color={0,0,127}));
+  connect(senMasFlo.m_flow,chiStaCon.mFloChiWat)
+    annotation (Line(points={{-180,-79},{-180,172},{-246,172},{-246,188},{-242,188}},color={0,0,127}));
+  connect(chiBypCon.y,valByp.y)
+    annotation (Line(points={{-59,-150},{-30,-150},{-30,-102}},color={0,0,127}));
+  connect(senMasFloByp.m_flow,chiBypCon.mFloByp)
+    annotation (Line(points={{30,-101},{30,-166},{-90,-166},{-90,-154},{-82,-154}},color={0,0,127}));
+  connect(chiStaCon.y,chiBypCon.chiOn)
+    annotation (Line(points={{-219,194},{-160,194},{-160,-147},{-82,-147}},color={255,0,255}));
+  annotation (
+    defaultComponentName="pla",
+    Documentation(
+      info="<html>
+<p>This model showcases a generic district central cooling plant as illustrated in the schematics below. </p>
+<ul>
+<li>The cooling is provided by two parallel chillers instantiated from </li>
```

Comment:
The end tag </li> should go all the way at the end of each bullet point, even if that means that it will be formatted on the same line on some display. If you want to do a line break, you should do it with a <br/> tag, but still keep everything that belong to the same bullet under the same <li>...</li> tag brace. Typically, I haven't seen any documentation that used a line break in the middle of a sentence, so you can just let them run very long.

Reply:
We agree, let’s just let them run very long.